src/Package/MSBuild.VSSetup/files.swr

```diff
@@ -48,6 +48,13 @@ folder InstallDir:\MSBuild\Current\Bin
   file source=$(X86BinPath)System.Runtime.CompilerServices.Unsafe.dll vs.file.ngenArchitecture=all
   file source=$(X86BinPath)System.Threading.Tasks.Dataflow.dll vs.file.ngenArchitecture=all vs.file.ngenPriority=1
   file source=$(X86BinPath)System.Collections.Immutable.dll vs.file.ngenArchitecture=all vs.file.ngenPriority=1
+  file source=$(X86BinPath)StreamJsonRpc.dll vs.file.ngenArchitecture=all
+  file source=$(X86BinPath)Nerdbank.Streams.dll vs.file.ngenArchitecture=all
+  file source=$(X86BinPath)System.IO.Pipelines.dll vs.file.ngenArchitecture=all
+  file source=$(X86BinPath)Microsoft.VisualStudio.Threading.dll vs.file.ngenArchitecture=all
+  file source=$(X86BinPath)Microsoft.VisualStudio.Validation.dll vs.file.ngenArchitecture=all
```

Comment:
I'm worried about this:

1. Does it work on .NET Core?
2. How do we ensure that in Visual Studio scenarios we're using the same version as VS (ideally the same physical file and we wouldn't have to redistribute)?

Reply:
1. This is dependency of StreamJsonRpc, but to your question, yes it should (https://www.nuget.org/packages/Microsoft.VisualStudio.Validation says it is nuget for .NET Standard). 
2. To be honest, I don't have any idea 😞

Reply:
Non of these newly added dependencies is loaded by denenv.exe. Tried it by starting instalation of VS with this version of MSBuild, building MSBuild project and checking output from WinDbg (thanks @ladipro for showing me how to do it). 
Only these DLLs were loaded from MSBuild folder in VS.
![image](https://user-images.githubusercontent.com/13519594/91821031-ad69c280-ec36-11ea-8319-871c9d5e8fa8.png)